### PR TITLE
Add gitter option in connect with us section

### LINKED ISSUE #335: Add giiter or IRC chat link in connect with us section
### Please describe the problem (or idea)
Introduce Gitter or IRC chat along other connect with us platforms
## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site. 

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

### PR DESCRIPTION
**Pull request by govindgoel**
Fixes #335 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] PR body includes `fixes #0000`-style reference to original issue #
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

**Comment by govindgoel** [comment]
@publiclab/reviewers  Please review this,changes look like this

File: examples/demo.css [hunk 1]
```diff
@@ -78,7 +78,7 @@ h1, h2, .section h3 {
     margin: auto;
     width: 80%;
     display: flex;
-    flex-wrap: wrap;
+    flex-wrap: nowrap;
```
Comment (cesswairimu) [inline examples/demo.css]: Hey @govindgoel, thanks for opening this...why did you change this to `nowarap`?

File: index.html [hunk 2]
```diff
@@ -273,7 +273,8 @@ <h2 class="text-center d-block">Connect with us</h2>
 		  <a class="col-md-2 links " href="https://www.facebook.com/PublicLab/" title="Facebook" ><i class="fa fa-facebook fa-3x"></i> </a>
 		  <a class="col-md-2 links " href="https://twitter.com/PublicLab?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor" title="Twitter" ><i class="fa fa-twitter fa-3x"></i> </a>
 		  <a class="col-md-2 links " href="https://github.com/publiclab" title="Organization" ><i class="fa fa-github fa-3x"></i> </a>
-		  <a class="col-md-2 links " href="https://publiclab.org" title="Publiclab Website"><i class="fa fa-globe fa-3x"></i></a>
+      <a class="col-md-2 links " href="https://publiclab.org" title="Publiclab Website"><i class="fa fa-globe fa-3x"></i></a>
+      <a class="col-md-2 links " href="https://gitter.im/publiclab/publiclab" title="Gitter"><i class="fa fa-comment fa-3x"></i></a>
```
Comment (cesswairimu) [inline index.html]: Also, could you please indent these. Thanks

File: examples/demo.css (hunk 1 above)
Comment (govindgoel) [reply examples/demo.css]: Wrap is something like this

Screenshot from 2020-02-16 22-57-07

**Comment by govindgoel** [comment]
@cesswairimu  Please review this
